<!DOCTYPE html>
<html lang="en">

    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <title>Software - Bradley T. Martin</title>
        <link rel="stylesheet" href="css/style.css">
    </head>

    <body>
        <header>
            <h1>Martin Lab</h1>
            <div class="index-item">
                <h2>Software</h2>
            </div>
        </header>
        <nav>
            <ul>
                <li><a href="index.html">Home</a></li>
                <li><a href="research.html">Research Interests</a></li>
                <li><a href="people.html">People</a></li>
                <li><a href="education.html">Education & Experience</a></li>
                <li><a href="publications.html">Publications</a></li>
                <li><a href="presentations.html">Conference Presentations</a>
                </li>
                <li><a href="software.html" class="active">Software</a></li>
                <li><a href="contact.html">Contact</a></li>
            </ul>
        </nav>
        <section>
            <h2>Research Software</h2>
            <div class="research-item">
                <img src="assets/images/geogenie-logo.png"
                    alt="GeoGenIE Software Logo" class="research-logo">
                <div>
                    <h3>GeoGenIE: Geographic-Genetic Inference Engine</h3>
                    <p>GeoGenIE (Geographic-Genetic Inference Engine) is an
                        advanced software tool designed to use deep learning for
                        predicting geographic localities (latitude and
                        longitude) based on genetic SNP data inputs. Built on
                        PyTorch, GeoGenIE is specifically optimized to account
                        for geographic sampling bias and the limited number of
                        SNPs typically found in GT-seq panels. This software is
                        a powerful resource for researchers who need to infer
                        geographic origins from genetic data, providing a robust
                        framework for handling complex datasets.</p>
                    <p>GeoGenIE not only excels in prediction accuracy compared
                        to other existing software, but also offers a wealth of
                        metrics and visualizations that allow users to
                        thoroughly assess model performance. The software is
                        highly user-friendly, ensuring that even those with
                        limited experience in deep learning can effectively
                        utilize its capabilities. GeoGenIE is also highly
                        efficient, with optional parallelized bootstrapping
                        across multiple CPU cores to speed up analysis. Users
                        have extensive flexibility in modifying model
                        parameters, visualization options, and preprocessing
                        settings, all while adhering to standard best practices.
                        GeoGenIE leverages vectorized Python modules such as
                        numpy, scipy, scikit-learn, and pandas, ensuring both
                        speed and reliability in processing.</p>
                </div>
            </div>
            <div class="research-item">
                <img src="assets/images/snpio-logo.png"
                    alt="SNPio Software Logo" class="research-logo">
                <div>
                    <h3>SNPio: Object-Oriented Python API for Population Genomic
                        Data Filtering, Visualization, and File Conversion</h3>
                    <p>SNPio is a cutting-edge software project aimed at
                        creating a user-friendly, object-oriented Python API
                        designed for reading, filtering, and converting various
                        standard genomic data file formats. In the complex field
                        of genomics, researchers often face challenges when
                        dealing with diverse data formats and the need for
                        efficient data manipulation tools. SNPio addresses these
                        challenges by providing a streamlined interface that
                        simplifies the management of genomic datasets, enabling
                        users to focus on analysis and interpretation.</p>
                    <p><a href="https://github.com/btmartin721/SNPio">SNPio
                            Source Code</a></p>
                    <p><a href="https://github.com/btmartin721/SNPio/issues">SNPio
                            Bug & Issue Tracking</a></p>
                    <p><a href="https://snpio.readthedocs.io/">SNPio
                            API Documentation</a></p>
                </div>
            </div>
            <div class="research-item">
                <img src="assets/images/pg-sui-logo.png"
                    alt="PG-SUI Software Logo" class="research-logo">
                <div>
                    <h3>PG-SUI: Population Genomics - Supervised and
                        Unsupervised Imputation</h3>
                    <p>PG-SUI (Population Genomics - Supervised and Unsupervised
                        Imputation) is an
                        AI-based software framework developed to address
                        the challenge of missing data in genomic SNP datasets,
                        which is common in
                        population genomics and can lead to biased results and
                        reduced statistical
                        power. PG-SUI employs both supervised and unsupervised
                        machine learning models
                        to accurately predict missing SNPs by analyzing
                        non-linear patterns in existing
                        data. The software is designed to enhance the accuracy
                        of genomic studies,
                        reduce the need for extensive data collection, and
                        integrate seamlessly into
                        bioinformatics pipelines, making it accessible to a
                        broader range of
                        researchers.</p>
                    <p><a href="https://github.com/btmartin721/PG-SUI">PG-SUI
                            Source Code</a></p>
                    <p><a href="https://github.com/btmartin721/PG-SUI/issues">PG-SUI
                            Bug & Issue Tracking</a></p>
                    <p><a href="https://pg-sui.readthedocs.io/">PG-SUI
                            API Documentation</a></p>
                </div>
            </div>
            <div class="research-item">
                <img src="assets/images/clinehelpr-logo.png"
                    alt="Cline Help R Software Logo" class="research-logo">
                <div>
                    <h3>ClineHelpR: Visualizing Genomic and Environmental Clines
                        using R</h3>
                    <p>ClineHelpR is an intuitive R package designed to
                        streamline the plotting and
                        visualization of Bayesian Genomic Cline (BGC) analyses
                        and results from the
                        INTROGRESS R package. Tailored for researchers working
                        with genomic clines,
                        ClineHelpR simplifies the detection of cline outliers
                        and facilitates the
                        correlation of environmental data with genomic clines
                        and hybrid indexes. With a
                        suite of user-friendly functions, this package empowers
                        users to generate clear,
                        publication-ready visualizations, making complex data
                        more accessible and
                        enhancing the interpretation of genomic and
                        environmental relationships in
                        hybrid zones.</p>
                    <p><a href="https://github.com/btmartin721/ClineHelpR"
                            target="_blank" rel="noopener noreferrer">ClineHelpR
                            Source Code</a>
                    </p>
                    <p><a href="https://github.com/btmartin721/ClineHelpR/issues"
                            target="_blank" rel="noopener noreferrer">ClineHelpR
                            Bug & Issue Tracking</a>
                    </p>
                </div>
            </div>
        </section>
        <footer>
            <p>&copy; 2024 Bradley T. Martin</p>
        </footer>
        <script src="js/script.js"></script>
    </body>

</html>